data/meson.build

```diff
@@ -0,0 +1,32 @@
+# icons
+icon_dir = join_paths(datadir, 'icons', 'hicolor')
+icon_basename = '@0@'.format(project_name)
+
+# scalable
+scalable = join_paths('icons', 'hicolor', 'scalable', 'apps', icon_basename + '.svg')
+install_data(scalable, install_dir: join_paths(icon_dir, 'scalable', 'apps'))
+
+# regular icons
+foreach size: ['16x16', '22x22', '24x24', '32x32', '48x48']
+  icon_name = join_paths('icons', 'hicolor', size, 'apps', icon_basename + '.png')
+  install_data(icon_name, install_dir: join_paths(icon_dir, size, 'apps'))
+endforeach
+
+
+po_dir = join_paths(meson.source_root(), 'po')
+# desktop file
+i18n.merge_file(input: '@0@.desktop.in'.format(app_id),
+                output: '@0@.desktop'.format(app_id),
+                install: true,
+                install_dir: join_paths(datadir, 'applications'),
+                po_dir: po_dir,
```

Comment:
You could do `po_dir: '../po'` instead of `join_paths(meson.source_root(), 'po')`.

Also `meson.source_root()` is deprecated in 0.56 in favor of `meson.project_source_root()`

Reply:
Personally I prefer being super explicit with things like this, when I remember to be consistent with it at least. I switched to `project_source_root()` though.

Reply:
Just kidding that's not supported by ubuntu...